ex1/ex1.js

```diff
@@ -6,3 +6,27 @@
  *     fn(",,,") === 2
  *     fn("hel,lowo,rld") === -1
  */
+
+/////////////////////////////////////////////////////////////////////////////
+// getThirdCommaIndex accepts a string and returns the index of the third comma in that string if no comma return -1
+function getThirdCommaIndex(str,searcFor) {
+
+        var index = -1;
+
+	for (let i = 0; i < 3; i++) {
+		index = str.indexOf(searcFor, index+1);
+		if (index == -1) { 
+			break;
+		}
+	}
+	return index;
+}
+
+function testFunction(str) {
+
+	console.log(getThirdCommaIndex(str, ','));
```

Comment:
Please switch to console.assert.
Then, read https://jasmine.github.io/ and switch to a real test framework (after which, running `npm test` should test your code and report if anything is wrong. Good luck